index.js

```diff
@@ -134,9 +135,33 @@ function getCoveragePercentage({ jobId, startTime, endTime }) {
             username: adminToken
         }
     })
-        .then(result => hoek.reach(result, 'measures.0.history.0.value'))
+        .then((result) => {
+            const metrics = {
+                coverage: hoek.reach(result, 'measures.3.history.0.value') || 'N/A',
+                tests: 'N/A'
+            };
+            const total = hoek.reach(result, 'measures.0.history.0.value');
+            const testErrors = hoek.reach(result, 'measures.1.history.0.value');
+            const testFailures = hoek.reach(result, 'measures.2.history.0.value');
```

Comment:
do these need default values too?

Reply:
I set `metrics.tests = N/A` by default already. if `total` has a value, `testErrors` and `testFailures` will always have a value, verified that by calling sonar api. And the case to cover valid `total` value is here: https://github.com/screwdriver-cd/coverage-sonar/blob/1aa9fcd7d0e933359e91314d7267e7d832142221/index.js#L147

Reply:
Exactly! Those are my concern when i look at the api response. Not nitpicky but two suggestions:

1) the order in which the requested metrics may be different than the order in the request query string. so `measures[0]` might correspond to `metric: tests`, in my case `[0]` is `tests_errors`.
2) if `total` is your control, it will make sense to only create const vars `testErrors` and `testFailures` inside the `if` since u r not using them anywhere else.